packages/apt/test.sh

```diff
@@ -181,7 +189,26 @@ fi
 
 echo "::endgroup::"
 
-if [ "${can_upgrade}" = "yes" ]; then
+if [ "${is_first_release}" = "no" ]; then
+  run_test
+fi
+
+echo "::group::Downgrade"
+
+if [ "${is_first_release}" = "no" ]; then
+  createdb downgrade
+  psql downgrade -c 'CREATE EXTENSION pgroonga'
+  psql downgrade -c \
+       "ALTER EXTENSION pgroonga UPDATE TO '${pgroonga_latest_released_version}'"
```

Comment:
`-c`はSQLとセットだから、`-c`とSQLを分断するのはやめようか。

```suggestion
  psql downgrade \
    -c "ALTER EXTENSION pgroonga UPDATE TO '${pgroonga_latest_released_version}'"
```